app/helpers/application_helper.rb

```diff
@@ -1,2 +1,10 @@
 module ApplicationHelper
+  BASE_TITLE = 'BIGBAG Store'.freeze
+  def full_title(page_title)
```

Comment:
引数にデフォルト値を設定するのはどうでしょうか? `BIGBAG Store`のみを表示したい時に現在の実装だと空文字(`""`)を渡す必要があるため、ここを改善できるかと思います。

```suggestion
  def full_title(page_title = "")
```

Reply:
空文字を渡すように修正いたしましたが、rubocopで キーワード引数を使用してくださいというエラーが出てしまう為、rubocop.ymlに以下のコードも追加しています。
Airbnb/OptArgParameters:
  Enabled: false